ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -4882,7 +4834,7 @@ components:
 
     NotifyPartyHBL:
       type: object
-      title: Notify Party
+      title: Notify Party (House B/L)
```

Comment:
Don't we need the condition - Mandatory if House B/L isToOrder=true here?

Reply:
I will ask business

Reply:
HAHAHHA - but we have - it is just "outside" GitHub view
<img width="634" alt="image" src="https://github.com/user-attachments/assets/5671d057-ce61-484a-ad84-a4ba335b90e6">